app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true, format: { with: VALID_EMAIL_REGEX }, uniqueness: true
+
+
+  def full_name
```

Comment:
please add tests for this method

Reply:
validates :first_name, last_name, email, presence: true
validates :email, format: { with: VALID_EMAIL_REGEX }, uniqueness: true